packages/ckeditor5-paste-from-office/src/filters/list.js

```diff
@@ -225,17 +225,23 @@ function findBulletedListStyle( element ) {
 // @param {module:engine/view/element~Element} element
 // @returns {module:engine/view/text~Text|null}
 function findListMarkerNode( element ) {
-	// If the first child is a text node, it is a value for the element.
+	// If the first child is a text node, it is the data for the element.
+	// The list-style marker is not present here.
 	if ( element.getChild( 0 ).is( '$text' ) ) {
 		return null;
 	}
 
-	const textNodeOrElement = element.getChild( 0 ).getChild( 0 );
+	// The list-style marker will be inside the `<span>` element. Let's ignore all non-span elements.
+	// It may happen that the `<a>` element is added as the first child. Most probably, it's an anchor element.
+	let textNodeOrElement = [ ...element.getChildren() ].filter( element => element.is( 'element', 'span' ) );
```

Comment:
We are doing a lot of unnecessary operations here:

* converting iterable to an array which removes benefits from having a generator
* iterating through all elements with filter method

to only get the first span child. Parsing Office content has good overload already, so maybe we could improve this code a bit performance-wise?

I've added a commit message with the proposed change as I can't use GH suggestions on deleted content. Please take a look if you agree to these changes @pomek

Reply:
Thanks for the proposed change. It makes sense. Let's wait for a response for CI, and if it still is green, let's merge the PR.

Reply:
As I can see, we're restarting one of the jobs because something fails, but it is not related to our work. I will verify the PR locally then we can merge it without waiting for CI results.